aten/src/ATen/native/native_functions.yaml

```diff
@@ -3158,7 +3158,7 @@
 - func: _trilinear(Tensor i1, Tensor i2, Tensor i3, int[] expand1, int[] expand2, int[] expand3, int[] sumdim, int unroll_dim=1) -> Tensor
   use_c10_dispatcher: full
 
-- func: triplet_margin_loss(Tensor anchor, Tensor positive, Tensor negative, float margin=1.0, float p=2, float eps=1e-06, bool swap=False, int reduction=Mean) -> Tensor
+- func: triplet_margin_loss(Tensor anchor, Tensor positive, Tensor negative, float margin=1.0, float p=2, float eps=1e-06, bool swap=False, int reduction=Mean, TODO? distance_function=None, bool is_similarity_function=False) -> Tensor
```

Comment:
Not sure what to put here, as it seems that native_functions doesn't support function args.  Or is there an alternate place to declare functions that would be C++ only?